mobidam-sst-management-backend/src/main/java/de/muenchen/mobidam/rest/ZuordnungController.java

```diff
@@ -0,0 +1,51 @@
+package de.muenchen.mobidam.rest;
+
+import de.muenchen.mobidam.domain.dtos.ZuordnungCreateDTO;
+import de.muenchen.mobidam.domain.dtos.ZuordnungDTO;
+import de.muenchen.mobidam.domain.mappers.ZuordnungMapper;
+import de.muenchen.mobidam.exceptions.SavingException;
+import de.muenchen.mobidam.service.ZuordnungService;
+import io.swagger.v3.oas.annotations.Operation;
+import jakarta.validation.Valid;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping(value = "/api/zuordnung")
+public class ZuordnungController {
+    private final ZuordnungService zuordnungService;
+    private final ZuordnungMapper zuordnungMapper;
+
+    @Operation(summary = "Create an zuordnung")
+    @PostMapping
+    public ResponseEntity<ZuordnungDTO> create(@Valid @RequestBody ZuordnungCreateDTO zuordnungCreateDTO) {
+        try {
+            ZuordnungDTO savedZuordnung = zuordnungService.create(zuordnungCreateDTO);
+            return new ResponseEntity<>(savedZuordnung, HttpStatus.OK);
+        } catch (SavingException savingException) {
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+        }
+    }
```

Comment:
Die ganze Geschichte mit dem Exception Handling würde ich in einen globalen Handler auslagern, dadurch dass wir voraussichtlich an mehrere Stellen die gleichen Exception-Arten haben werden.

Beispielimplementierung: https://medium.com/@aedemirsen/spring-boot-global-exception-handler-842d7143cf2a

```suggestion
    @Operation(summary = "Create an zuordnung")
    @PostMapping
    public ResponseEntity<ZuordnungDTO> create(@Valid @RequestBody ZuordnungCreateDTO zuordnungCreateDTO) {
          return new ResponseEntity<>(zuordnungService.create(zuordnungCreateDTO), HttpStatus.OK);
    }
```